### PR TITLE
Full ZTP flow with spoke cluster installation

### PR DESCRIPTION
**Pull request by osherdp**
Changed ztp flow to wait for a complete spoke cluster installation.
Also, we'll make use of code from assisted-service instead of code defined on ``openshift/release``
/cc @YuviGold

### CODE REVIEW

File: ci-operator/step-registry/baremetalds/assisted/operator/ztp/baremetalds-assisted-operator-ztp-commands.sh [hunk 1]
```diff
@@ -10,6 +10,10 @@ echo "************ baremetalds assisted operator ztp command ************"
 # shellcheck source=/dev/null
 source "${SHARED_DIR}/packet-conf.sh"
 
+git clone https://github.com/osherdp/assisted-service --branch feature/ztp-installation-for-sno
```
Comment (osherdp) [inline ci-operator/step-registry/baremetalds/assisted/operator/ztp/baremetalds-assisted-operator-ztp-commands.sh:13]: ignore, will get removed once it's pushed

**Review by YuviGold — APPROVED** [review]

**Pull request merged**